src/blocked.c

```diff
@@ -464,7 +464,7 @@ void serveClientsBlockedOnStreamKey(robj *o, readyList *rl) {
                 if (group) {
                     noack = receiver->bpop.xread_group_noack;
                     sds name = receiver->bpop.xread_consumer->ptr;
-                    consumer = streamLookupConsumer(group,name,SLC_DEFAULT);
+                    consumer = streamLookupConsumer(group,name);
```

Comment:
i understand we update the active time in streamReplyWithRange, but maybe we wanna update seen time as well (when a blocked client gets unblocked)?

Reply:
idk, seen_time suggests an active attempt (in my eyes, it should only be updated by a direct client interaction)

Reply:
If I may chime in (I'm one of the user who requested this change): I agree with the notion that seen time should represent the point in time when the last request of a consumer was received. 

Our use case is that we want to estimate when a consumer was proven to be ‘alive’ for the last time. This is the case when a request was received from it. When a consumer unblocks, it is not necessarily ‘alive’. It could also be dead and the TCP connection just hasn't noticed that yet.

Reply:
@oranagra can we resolve this?